MAD.Extensions.EFCore/UpsertExtensions.cs

```diff
@@ -11,32 +11,51 @@ namespace MAD.Extensions.EFCore
 {
     public static class UpsertExtensions
     {
-        public static void Upsert(this DbContext dbContext, object entity, Action<object> transformations = null)
```

Comment:
Please add back the transformations action, otherwise this is a breaking change.

Reply:
Transformations added back in